plugins/include/amxmodx.inc

```diff
@@ -1391,7 +1391,7 @@ native get_playersnum(flag = 0);
  *
  * @noreturn
  */
-native get_players(players[MAX_PLAYERS], &num, const flags[] = "", const team[] = "");
+native get_players(players[MAX_PLAYERS] = 0, &num, const flags[] = "", const team[] = "");
```

Comment:
I think `{}` should be better.

Reply:
noticed plugin size got smaller

Reply:
had to read it a couple of times to understand it, but its good that we got rid of the buffer creation and copying operations but is editing the get_players function in amxmodx.cpp to check if the first param is a valid array will be better then current code ?

Reply:
It's not possible to do that (unless we introduce something like `NULL_ARRAY`) as far as I know. The only solution would be to add a new parameter, but in the end it doesn't matter - the native has to do the filtering anyway, copying to the array and increasing the counter is trivial.

Reply:
@rsKliPPy i see, well its trivial but in some cases this may be called a lot in small delay tasks for example, i guess another native in amxmodx.cpp is the best option if we want to do the most optimized way